library/lib/apollon-editor.tsx

```diff
@@ -50,6 +50,7 @@ export class ApollonEditor {
       this.diagramStore,
       this.metadataStore
     )
+    this.diagramStore.getState().initializeUndoManager()
```

Comment:
This should be before the rendering. When I was testing in give feedback, see feedback, I was able to remove all the diagram data. 

```suggestion

    if(this.metadataStore.getState().mode === Apollon.ApollonMode.Modelling) {
          this.diagramStore.getState().initializeUndoManager()
    }

```